src/migrations/cluster-store/3.6.0-beta.1.ts

```diff
@@ -23,69 +23,68 @@
 // convert file path cluster icons to their base64 encoded versions
 
 import path from "path";
-import { app, remote } from "electron";
+import { app } from "electron";
```

Comment:
are changes in this file related to deleting a cluster from the kubeconfig?

Reply:
Because I removed `kubeConfig: string;` from the `ClusterModel` I made `Pre360ClusterModel`.

The rest of the changes are releated to the fact that I removed `saveToAppFiles` as this is the only place that the `sync`-ness is needed and other places should use the promise based versions.